gatling-app/src/main/scala/io/gatling/app/RunResultProcessor.scala

```diff
@@ -68,7 +68,7 @@ private final class RunResultProcessor(configuration: GatlingConfiguration) {
       assertionResult match {
         case AssertionResult.Resolved(_, success, actualValue) =>
           if (success) {
-            println(s"$message : true")
+            println(s"$message : true (actual : $actualValue)")
           } else {
             println(s"$message : false (actual : $actualValue)")
           }
```

Comment:
Now, same thing as:
```suggestion
          println(s"$message : $success (actual : $actualValue)")
```

Pedantic note: no space before semicolon in english:
```suggestion
          println(s"$message: $success (actual: $actualValue)")
```